bin/applab-data-stats

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env ruby
+
+require 'aws-sdk'
+require_relative '../deployment'
+require 'cdo/db'
+require_relative '../shared/middleware/helpers/storage_id'
+
+db = Aws::DynamoDB::Client.new
+
+last_evaluated_key = nil
+hash_map = {}
+channel_map = {}
+rows = 0
+consumed_capacity = 0
+
+begin
+  response = db.scan(
+      table_name: CDO.dynamo_tables_table,
+      attributes_to_get: ['channel_id', 'hash'],
+      exclusive_start_key: last_evaluated_key,
+      return_consumed_capacity: 'TOTAL'
+  )
+  consumed_capacity += response.consumed_capacity.capacity_units
+  page = response.first
+
+  page[:items].each do |item|
+    channel_map[item['channel_id']] = 1
+    hash_map[item['hash']] = 1
+    rows += 1
+  end
+
+  last_evaluated_key = page[:last_evaluated_key]
+end while last_evaluated_key
+
+$stderr.puts "tables: #{hash_map.keys.length}"
+$stderr.puts "channels: #{channel_map.keys.length}"
+$stderr.puts "rows: #{rows}"
+$stderr.puts "1st pass consumed_capacity: #{consumed_capacity}"
```

Comment:
The cleaner way to do this might be to provide a file as an input, and explicitly write to that (with logging going to stdout), but I think this script is enough of a one-off that it's probably not worth taking the time to change what we have/have used/have shown to work.

That said, it's probably worth a short comment at the top of the file summarizing how we expect to call this file, and when we might want to use it.